astropy/io/fits/tests/test_table.py

```diff
@@ -545,7 +545,7 @@ def test_appending_a_column(self):
         for i in range(len(t1[1].columns)):
             hdu.data.field(i)[nrows1:] = t2[1].data.field(i)
 
-        hdu.writeto(self.temp('newtable.fits'))
+        hdu.writeto(self.temp('newtable.fits'), overwrite=True,)
```

Comment:
I guess you needed this for debugging, but could you remove it now ?

Reply:
yes